include/ts/apidefs.h.in

```diff
@@ -880,6 +880,15 @@ typedef enum {
   TS_MGMT_SOURCE_ENV       ///< Process environment variable.
 } TSMgmtSource;
 
+/// The User Arg type, used for Txn/Ssn/VConn user argument slots
```

Comment:
This might be problematic. Because it's C (and because in C++ it's not an `enum class`) these names are injected in to
the surrounding (global) scope. It would probably be better to prefix the enumeration names to reduce collisions. E.g.
```
typedef enum {
   USER_ARG_TXN,
   USER_ARG_SSN,
// ....
} TSUserTypeType;
```

Reply:
Yes, an oversight on my part, thanks for spotting. And for consistency, they should be prefixed TS_.